ehitchhiking-ui/src/app/shared/components/trips-modal/trips-modal.component.html

```diff
@@ -10,14 +10,31 @@
   <mat-expansion-panel>
     <mat-expansion-panel-header>Filters</mat-expansion-panel-header>
     <div class="form-row expansion-panel-sm">
-      <mat-form-field>
+      <mat-form-field class = "filter-width">
         <mat-label>Role</mat-label>
         <mat-select [(ngModel)]="selectedRole" name="selected">
           <mat-option (click)="filterByRole()" *ngFor="let role of roles" [value]="role.value">
             {{role.viewValue}}
           </mat-option>
         </mat-select>
       </mat-form-field>
+
+      <mat-form-field class = "filter-width">
+        <mat-label>Status of trip</mat-label>
+        <mat-select [formControl]="statuses" multiple>
+          <mat-option (click)="filterByStatus()" *ngFor="let status of statusesTrip"
+                      [value]="status.value">{{status.viewValue}}</mat-option>
+        </mat-select>
+      </mat-form-field>
+
+      <mat-form-field class = "filter-width">
+        <mat-label>rating of trip</mat-label>
```

Comment:
Just "Rating" instead of "rating of trip"

Reply:
okay